x-pack/plugins/maps/public/connected_components/edit_layer_panel/layer_settings/layer_settings.tsx

```diff
@@ -231,6 +231,7 @@ export function LayerSettings(props: Props) {
         {renderZoomSliders()}
         <AlphaSlider alpha={props.layer.getAlpha()} onChange={onAlphaChange} />
         {renderShowLabelsOnTop()}
+        <EuiSpacer size="m" />
```

Comment:
Is this spacer change still needed now that the editor has been moved to "Layer style" panel?